### PR TITLE
Patmos docs updated to reflect lf-lang/lingua-franca#2383

### PR DESCRIPTION
**Pull request by EhsanKhodadad**
See https://github.com/lf-lang/reactor-c/pull/383, which is yet to be completed and merged.

### CODE REVIEW

**Comment by lhstrh** [comment]
The PR that needs to get merged before these docs are relevant is seems to be https://github.com/lf-lang/lingua-franca/pull/2383.

**Review by lhstrh — COMMENTED** [review]
I'm not a user of the Patmos platform, so it's hard for me to review these docs. Maybe @schoeberl can help? Having to create a Makefile among the generated sources strikes me as problematic... The whole build workflow is based on CMake, and this seems to bypass that completely. Why? @erlingrj has worked on platform support prior to this, so let me tag him here as well.

File: docs/embedded/patmos.mdx [hunk 1]
```diff
@@ -7,101 +7,121 @@ Lingua Franca's C-runtime supports [Patmos](https://github.com/t-crest/patmos),
...
 
-To execute reactions on Patmos, the [Patmos toolchain](https://github.com/t-crest/patmos) needs
-to be installed. The web page contains a quick start, detailed information including how to
-perform WCET analysis is available in the
-[Patmos Reference Handbook](http://patmos.compute.dtu.dk/patmos_handbook.pdf).
+    make -C src-gen/HelloWorld
 
-To execute the "hello world" reactor on Patmos use the LF compiler to generate the C code.
-Compile the reactor with the Patmos compiler (in `src-gen`):
+Since LF still doesn't support Patmos officialy, you need to copy some files inluding `lf_patmos_support` c and h files in the related folders before executing make command whether manually or by executing the following shell file that automates the copying process (considering those files are located in a folder called `files`). If you choose shell file, you can also add Makefile to the list.
```
Comment (lhstrh) [inline docs/embedded/patmos.mdx:90]: I don't believe this is still accurate?

Comment (schoeberl) [reply docs/embedded/patmos.mdx:90]: Yes, probably this is not anymore accurate.

**Review by lhstrh — APPROVED** [review]
Looks great! Thanks, @EhsanKhodadad.

**Pull request merged**